roles/openshift_openstack/templates/heat_stack.yaml.j2

```diff
@@ -540,7 +563,7 @@ resources:
       name: infra_server_group
       policies: {{ openshift_openstack_infra_server_group_policies }}
 {% endif %}
-{% if openshift_openstack_num_masters|int > 1 %}
+{% if openshift_openstack_use_vm_load_balancer %}
```

Comment:
This changes the default behavior for multiple masters. With `openshift_openstack_use_vm_load_balancer: false` by default, there would be no more loadbalancer provisioned for openshift-ansible multi-master out-of-box solution.  Is that what we want?

Reply:
Yes. You should opt-into a load balancer and when doing so, choose which one to use.

This also lets you use an external LB without wasting resources.

Reply:
Out of curiosity, what will happen if multiple masters are specified without a load balancer? Is that something we want to check against?

Reply:
The cluster would work just fine. You could talk to either of the master nodes to access the API or UI and similarly to either of the infra nodes for the public routes.

I've added an explanation to the docs.